pages/[user]/[type].tsx

```diff
@@ -87,13 +94,23 @@ export default function Type(props) {
                 times.splice(i, 1);
             }
         });
+
+      // If event ends after endTime, remove the slot
+      if(dayjs(times[i]).add(props.eventType.length, 'minute') > dayjs(times[i]).hour(0).minute(0).add(props.user.endTime, 'minute') ) {
```

Comment:
You may have fixed a bug here, but this shouldn't be necessary given earlier for() logic, it's unrelated to timeZone so I wonder what behaviour you fixed here.

Reply:
It is to avoid overtime booking. It accounts for the duration of the meeting.

If I set the end of my day at, let's say, 7 pm and the eventType has a duration of an hour, I don't want people to be able to book after 6 pm.

Maybe I misunderstood the meaning of the end of the day. This makes Calendso behave like Calendly.

Open to changes!